server/src/stacks/2020-10-01/stacks/function-app-stacks/Java.ts

```diff
@@ -25,6 +25,7 @@ const getJavaStack: (useIsoDateFormat: boolean) => FunctionAppStack = (useIsoDat
                 isPreview: false,
                 isHidden: false,
                 isAutoUpdate: true,
+                 isDefault: true,
```

Comment:
```suggestion
                isDefault: true,
```
nit: remove the extra space.